api/types/types.go

```diff
@@ -461,13 +461,23 @@ type Runtime struct {
 	Args []string `json:"runtimeArgs,omitempty"`
 }
 
+// DownloadCacheUsage contains information about the disk usage of the download
+// cache
+type DownloadCacheUsage struct {
+	TotalNumber  int64
+	TotalBytes   int64
+	ActiveNumber int64
+	ActiveBytes  int64
+}
+
 // DiskUsage contains response of Engine API:
 // GET "/system/df"
 type DiskUsage struct {
-	LayersSize int64
-	Images     []*ImageSummary
-	Containers []*Container
-	Volumes    []*Volume
+	LayersSize         int64
+	Images             []*ImageSummary
+	DownloadCacheUsage DownloadCacheUsage
```

Comment:
Should this be folded into the `ImageSummary`? It is odd for this to be separate (also for `LayersSize`, but can't undo that).